docs/developer/testing.md

```diff
@@ -4,7 +4,7 @@
 
 The entrypoint for testing any integration is the command `ddev test`, which accepts an arbitrary number of integrations as arguments.
 
-Under the hood, we use [tox][tox-github] for environment management and [pytest][pytest-github] as our test framework.
```

Comment:
Can we also remove the link to `[tox-github]`?

Reply:
[Not yet](https://github.com/DataDog/integrations-core/blob/d865720a65a01fe79c2d38f748b93120f47705a0/docs/developer/meta/docs.md?plain=1#L64) unfortunately :(

Florent says we have a jira card to update that stuff as well, since we actually do use tox to build the docs still.